packages/react/src/profile/__tests__/useProfileActionHistory.spec.ts

```diff
@@ -8,14 +8,13 @@ import { waitFor } from '@testing-library/react';
 import { setupHookTestScenarioWithSession } from '../../__helpers__/setupHookTestScenarioWithSession';
 import { useProfileActionHistory } from '../useProfileActionHistory';
 
-// TODO make it work
-describe.skip(`Given the ${useProfileActionHistory.name} hook`, () => {
```

Comment:
thanks for fixing it, I remember I couldn't figure out why it didn't want to work